TrustModel_Personas.md

```diff
@@ -0,0 +1,179 @@
+the Application or Workload which in turn provides value to the data owners (customers and clients).
+The workload provider and data owner could be part of same company/organisation but 
+following the least privilege principle the application provider should not be able to view or 
+manipulate end user data without informed consent.
+- Owner of business logic containers. These may contain proprietary algorithms, models or secrets
+- Need to prove to customer aspects of compliance.
+- Defines what the solution requires in order to run and maintain compliance (resources, utility 
+  containers/services,  storage).
+- Defines the method of verifying the container images to be used and provides artifacts to 
+  allow verification to be completed within the TEE.
+- provide the boot images initially required by the TEE during 
+  initialisation or designates a trusted party to do so.
+- provide the attestation verification service, or designate a trusted party to provide the 
+  attestation verification service.
+
+Example : 3rd party software vendor, cloud solution provider
+
+### Container Image Provider
+
+This persona is responsible for the part of the supply chain that builds container images and 
+provides them for use by the solution. Since a workload can 
+be composed of multiple containers, there may be multiple container image providers, some 
+will be closely connected to the workload provider (business logic containers), others more 
+independent to the workload provider (side car containers). The container image provider is expected
+to use a mechanism to allow provenance of container image to be established when a 
+workload pulls in these images at deployment time. This can take the form of signing or encrypting 
+the container images.
+- Builds container images
+- Signs or encrypts the images
+- Provides the signature or public encryption keys securely to the workload provider by a trusted 
+  channel.
+- Protects the keys used to sign or encrypt the container images
+
+It is recognised that hybrid options exist surrounding workload provider and container provider. 
+For example the workload provider may choose to protect their supply chain by 
+signing/encrypting their own container images after following the build patterns already 
+established by the container image provider. 
+
+Example : Sidecar Container Provider
+
+
+### Data Owner
+Owner of data used, and manipulated by the application.
+- concerned with visibility and integrity of their data.
+- concerned with compliance and protection of their data.
+- uses and shares data with solutions. 
+- least privilege principle means no visibility or manipulation of data is possible by 
+  Orchestration Operator or Cloud Operator personas.
+
+## Conclusion
+
+The key trust relationship here is between the Data Owner and the other personas. The Data Owner 
+trusts the code provided by the workload provider to operate across their data, however they do 
```

Comment:
with the `container image providers` the code is now provided by them?

Reply:
:-) Good catch I added Container Image Provider but didn't consider the impact on the conclusion section. I updated a few things in this section to fix what you found above but also highlight the trust between the workload provider and the container image provider.